carsus/util/colored_logger.py

```diff
@@ -0,0 +1,66 @@
+import logging
```

Comment:
This is a copy of `tardis/util/colored_logger.py`. This way we don't need `tardis` as a dependency of `carsus`.